### DISCUSSION TITLE
Allow access to relation item when disable editing

### DISCUSSION THREAD

**Discussion by TheAzack9** [directus/directus#4753] [Feature Requests]
Not sure whether to file this as an idea or possibly a bug 😅 It seems like it has been disabled intentionaly, but i am not sure...

I have not tested O2M, M2O or M2A, but at least with M2M if you create a relation, add items and then disable editing on the collection, then everything is greyed out. It would be very handy to be able to access the item as usual, but not be able to edit the relation.

In my head i would expect it to allow me to access the item and edit anything in the item normally. (that is not set to disable editing or because of other settings cannot be edited ofc) However anything regarding the relation (adding existing item, creating new item, reordering, removing) should be disabled. As it is now does it seem like i can change ordering, but i guess that is only visual and not saved... 

![image](https://user-images.githubusercontent.com/2140448/113001705-5c409380-9171-11eb-9564-cc9035f269d0.png)

![image](https://user-images.githubusercontent.com/2140448/113001785-6e223680-9171-11eb-8ec4-5e8772150c13.png)

**Comment by rijkvanzanten** [comment]
> In my head i would expect it to allow me to access the item and edit anything in the item normally. 

Interesting thought! As of right now, that wouldn't be technically possible, as the update is literally nested within the parent item when it's submitted, so if the parent item is disabled, it won't allow nested updates either 🤔

**Reply by TheAzack9** [reply]
Hmm, fair enough 🤔

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
